mkdocs.yml

```diff
@@ -82,6 +82,13 @@ pages:
     - Controller API v2.0: reference-guide/controller-api/v2.0.md
     - Controller API v2.1: reference-guide/controller-api/v2.1.md
     - Controller API v2.2: reference-guide/controller-api/v2.2.md
+  - Changelogs:
+    - v2.0.0: changelogs/v2.0.0.md
+    - v2.1.0: changelogs/v2.1.0.md
+    - v2.2.0: changelogs/v2.2.0.md
+    - v2.3.0: changelogs/v2.3.0.md
+    - v2.4.0: changelogs/v2.4.0.md
```

Comment:
can you also add 2.4.2. https://gist.github.com/kmala/3cfeb97c3d2ce2274c908b46ab316898